src/Host/Client/Impl/Host/BrokerClient.cs

```diff
@@ -95,6 +95,19 @@ internal abstract class BrokerClient : IBrokerClient {
             }
         }
 
+        public async Task<bool> DeleteRemoteUserProfileAsync(CancellationToken cancellationToken) {
```

Comment:
It's not remote if it's called on a local broker :) I think this should just be `DeleteProfileAsync`.